surefire-its/src/test/resources/cucumber-tests/src/test/java/cz/fafejta/RunCucumberTest.java

```diff
@@ -0,0 +1,14 @@
+package cz.fafejta;
+
+import org.junit.platform.suite.api.*;
+
+import static io.cucumber.junit.platform.engine.Constants.GLUE_PROPERTY_NAME;
+import static io.cucumber.junit.platform.engine.Constants.JUNIT_PLATFORM_NAMING_STRATEGY_PROPERTY_NAME;
+
+@Suite
+@IncludeEngines("cucumber")
+@SelectPackages("cz.fafejta")
+@ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "cz.fafejta")
+@ConfigurationParameter(key = JUNIT_PLATFORM_NAMING_STRATEGY_PROPERTY_NAME, value = "long")
```

Comment:
@olamy this parameter overrides `cucumber.junit-platform.naming-strategy=surefire` in the `pom.xml`. Is that intentional?

Reply:
I didn't pay attention as I picked one of the projects offered as an example of surefire not working anymore,
This doesn't look to have any effect. My only need was to have a project not working correctly with 3.5.3 and fix with this PR.

Reply:
Then you can remove `cucumber.junit-platform.naming-strategy=surefire ` from the `pom.xml`. The `surefire` strategy is a workaround that makes the naming algorithm pre-3.5.2 work nicely.